src/pages/taskHistory/ColumnHeaders.test.tsx

```diff
@@ -0,0 +1,176 @@
+import { MockedProvider, MockedResponse } from "@apollo/client/testing";
+import { context } from "components/HistoryTable";
+import { HistoryTableReducerState } from "components/HistoryTable/historyTableContextReducer";
+import {
+  fireEvent,
+  renderWithRouterMatch as render,
+  waitFor,
+} from "test_utils";
+import { string } from "utils";
+import ColumnHeaders from "./ColumnHeaders";
+
+const { trimMiddleText } = string;
+const { HistoryTableProvider } = context;
+const longVariantName = "really_really_really_really_really_long_variant_name";
+const trimmedVariantName = trimMiddleText(longVariantName, 50, 15);
+
+const initialState: HistoryTableReducerState = {
+  loadedCommits: [],
+  processedCommits: [],
+  processedCommitCount: 0,
+  commitCache: new Map(),
+  currentPage: 0,
+  pageCount: 0,
+  columns: [],
+  historyTableFilters: [],
+  commitCount: 10,
+  visibleColumns: [],
+  columnLimit: 7,
+};
+
+interface wrapperProps {
+  children: React.ReactNode;
+  mocks?: MockedResponse[];
+  state?: Partial<HistoryTableReducerState>;
+}
+
+const wrapper: React.FC<wrapperProps> = ({ children, mocks = [], state }) => (
+  <MockedProvider mocks={mocks}>
```

Comment:
Is the MockedProvider actually needed? it doesn't seem like we are actually mocking any queries.

Reply:
Ah no it's not needed 🙂 removing